src/client/helpers/postingMetadata.js

```diff
@@ -0,0 +1,29 @@
+import { has, forOwn, isEmpty } from 'lodash';
+
+const postingMetadataHelper = (jsonMetadata, postingJsonMetadata) => {
+  if (!isEmpty(jsonMetadata) && isEmpty(postingJsonMetadata)) {
```

Comment:
you can add local vars
const isJsonMetadata = !isEmpty(jsonMetadata);
const isPostingMetadata = !isEmpty(postingJsonMetadata);
and use them in conditions